src/pages/wallet-import-keystore/wallet-import-keystore.ts

```diff
@@ -193,4 +264,55 @@ export class WalletImportKeystorePage {
     return true;
   }
 
-}
+  importKeystoreFile() {
+    var input = document.createElement('input');
+    input.type = 'file';
+    input.click();
+    input.onchange = (e) => {
```

Comment:
I somewhat understand what is going on but still I think this code could use some documentation.

Reply:
Documentation added next request